client.go

```diff
@@ -347,3 +348,16 @@ func (uc *Client) getStrategy(name string) strategy.Strategy {
 func (uc *Client) WaitForReady() {
 	<-uc.onReady
 }
+
+// ListFeatures returns all available features toggles.
+func (uc *Client) ListFeatures() []*feature.Feature {
+	var features []*feature.Feature
+	for _, f := range uc.repository.list() {
+		features = append(features, &feature.Feature{
+			Name:         f.Name,
+			Description:  f.Description,
+			Enabled:      f.Enabled,
+		})
```

Comment:
should also include all details (strategy and variant details missing)

Reply:
Would you like *all* fields to be copied over?

My intuition was to not expose the `api` data structures but this is a lot of duplication now.